app/src/components/Empty/index.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cn from "classnames/bind";
+
+import style from "./empty.scss";
+
+const cx = cn.bind(style);
+
+const Empty = ({ errorMessage }) => {
+  return (
+    <div className={cx("empty-page")}>
+      <h2>Welcome to SIGHT.</h2>
+      <div>
+        <p>
+          You arrived here because no market is selected. Please visit
+          <a href="https://sight.pm/#markets" rel="noreferrer noopener">
+            {" "}
+            our market list{" "}
+          </a>
+          and select a market to continue.
+        </p>
+        <p>If you have any doubts you can contact us on the channels below:</p>
```

Comment:
"If you have any questions"? Or was this in the design?

Reply:
Just a proof of concept. Make sense to update